helpers/getSymbolDescription.js

```diff
@@ -5,26 +5,34 @@ var GetIntrinsic = require('../GetIntrinsic');
 var callBound = require('./callBound');
 
 var $SyntaxError = GetIntrinsic('%SyntaxError%');
+var getGlobalSymbolDescription = GetIntrinsic('%Symbol.keyFor%', true) || function () {};
```

Comment:
why the function fallback in this case? i'd prefer it just remain falsy if it's not present, and the implementation can if-check it at runtime.

Separately, I don't know why we'd want to involve the global registry at all - all symbols print out the description inside toString, and adding another method this depends on increases the API surface.

Reply:
Well, using `Symbol.keyFor` makes it possible to return `""` even in cases where `getInferredName` and the `Symbol.prototype.description` getter aren't supported for global symbols.

Reply:
Any idea in which engines this makes a difference?